backend/.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+    "env": {
+        "browser": true,
```

Comment:
might need to change this to "node": true since express app live in node.js otherwise it complains on app.js (it does on my editor). [click here for reference](/https://eslint.org/docs/user-guide/configuring/language-options#specifying-environments)

Reply:
look at eslint config file in uofablueprint.org repo, I'm getting a bunch of eslint error in my editor